controllers/v1/events/events.controller.js

```diff
@@ -25,7 +25,6 @@ const get = async (request, response) => {
 
   try {
     const events = await eventsService.doList(eventParameters)
-
     responseCode = events.responseCode
     responseData = baseController.getSuccessResponse(events.data, events.message)
```

Comment:
Teniendo en cuenta el comment anterior, recordemos siempre el principio DRY (Do not repeat yourself)
```suggestion
    responseCode = responseCode
    responseData = baseController.getSuccessResponse(data, message)
```